pkg/client/unversioned/persistentvolumeclaim.go

```diff
@@ -25,6 +25,10 @@ import (
 	"k8s.io/kubernetes/pkg/watch"
 )
 
+const (
+	persistentVolumeClaimResourceName string = "persistentVolumeClaims"
```

Comment:
"persistentVolumeClaims" was the old string used in invocations. Is that one wrong?